tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "noImplicitThis": true,
     "noImplicitAny": true,
     "noUnusedLocals": true,
-    "importHelpers": true,
+    "importHelpers": false,
```

Comment:
I was running into an issue where, with it, the imports from ThreadAction.ts would work either for the TS compiler or for Jest, but not both (i.e. we could typecheck or test, but both weren't working). I don't understand why this fixes it, but since we're not deploying to browsers, it also seems innocuous.

Reply:
Sorry, ThreadData imports not ThreadAction.